src/AccordionItemBody/AccordionItemBody.wrapper.js

```diff
@@ -20,29 +28,19 @@ class AccordionItemBodyWrapper extends Component<
         hideBodyClassName: 'accordion__body--hidden',
     };
 
-    renderItemBody = (
-        accordionStore: AccordionContainer,
-        itemStore: ItemContainer,
-    ) => {
-        const { uuid } = itemStore.state;
-        const { items, accordion } = accordionStore.state;
-        const item = items.filter(stateItem => stateItem.uuid === uuid)[0];
-        return (
-            <AccordionItemBody
-                {...this.props}
-                {...item}
-                accordion={accordion}
-            />
-        );
-    };
-
     render() {
-        return (
-            <Subscribe to={[AccordionContainer, ItemContainer]}>
-                {this.renderItemBody}
-            </Subscribe>
-        );
+        const { itemStore, accordionStore, ...rest } = this.props;
+        const { uuid } = itemStore;
+        const { items, accordion } = accordionStore;
+        const item = items.filter(stateItem => stateItem.uuid === uuid)[0];
```

Comment:
so not `.find` because that would require a polyfill?

Reply:
Exactly - no IE support for Array.prototype.find.